### PR TITLE
Ts lint fixes

### PR DESCRIPTION
**Pull request by ktw1016**

### CODE REVIEW

File: client/src/general_utils.ts [hunk 1]
```diff
@@ -3,6 +3,8 @@ import JSURL from "jsurl";
 import _ from "lodash";
 import marked from "marked";
 
+/* eslint-disable @typescript-eslint/no-explicit-any */
```
Comment (ktw1016) [inline client/src/general_utils.ts:6]: `any` in this file for `this` keyword. Not sure how to make decorators type safe

**Review by Stephen-ONeil — CHANGES_REQUESTED** [review]
Pretty minor things mostly

File: client/src/charts/graphRegistry.ts [hunk 2]
```diff
@@ -7,43 +7,46 @@ import { Dispatch, dispatch } from "d3-dispatch";
...
   height?: number;
   alternative_svg?: string;
-  dispatch?: Dispatch<any>;
+  dispatch?: Dispatch<T>;
   events?: string[];
 }
 
-class GraphRegistry {
-  registry: GraphRegistry[];
+class GraphRegistry<T extends Record<string, unknown>> {
```
[the remark this answers is not in the document]
Comment (ktw1016) [reply client/src/charts/graphRegistry.ts:17]: 👍

File: client/src/components/DisplayTable/DisplayTable.tsx [hunk 3]
```diff
@@ -35,10 +35,15 @@ interface ColumnKeyProps {
   is_summable: boolean; // Default to false
   is_searchable: boolean; // Default to false
   initial_visible: boolean; // Default to trues
-  formatter: string | Function; // If string, supply format key (e.g.: "big_int") found in format.js. If function, column value is passed in (e.g.: (value) => <span<{value}</span>)
-  raw_formatter: (val: any) => string; // Actual raw value for data. Value from this is used for sorting/searching/csv string. Default to _.identity. (e.g.: (value) => Dept.lookup(value).name)
-  sum_func: Function; // e.g.: (sum, value) => ... Default to sum + value
-  sort_func: Function; // e.g.: (a,b) => ... Default to _.sortBy
+  formatter: // If string, supply format key (e.g.: "big_int") found in format.js. If function, column value is passed in (e.g.: (value) => <span<{value}</span>)
+  string | ((val: string | number | Date) => string | React.ReactNode);
```
[the remark this answers is not in the document]
Comment (ktw1016) [reply client/src/components/DisplayTable/DisplayTable.tsx:39]: Yup I absolutely should

**Review by Stephen-ONeil — APPROVED** [review]
:+1:

**Pull request merged**